cobald_tests/controller/test_relative_supply.py

```diff
@@ -5,19 +5,15 @@
 
 
 class TestRelativeSupplyController(object):
-    def test_low_scale(self):
+    def test_parameter_verification(self):
```

Comment:
I combined these three test cases into one as before reading all of them it wasn't clear to me what each one tested. The new name hopefully makes this clearer.